src/io/flutter/editor/FilteredIndentsHighlightingPass.java

```diff
@@ -302,6 +302,17 @@ private long nowStamp() {
     return myDocument.getModificationStamp();
   }
 
+  public static void cleanupHighlighters(Editor editor) {
```

Comment:
handles edge case when regular indent guides are disabled but a source file hasn't been closed and reopened yet.